Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,26 @@
+.white-btn {
+  text-align: center;
+  line-height: 100px;
+  font-size: 50px;
+  width: 116.6px;
+  background-color: white;
+}
+
```

Comment:
Styling all these different elements widths would be much easier using % instead of px.